pkg/plugins/runtime/gateway/gateway_route_generator_test.go

```diff
@@ -1440,30 +1440,123 @@ conf:
   - port: 8080
     protocol: HTTP
     crossMesh: true
+    tags:
```

Comment:
I'd like to avoid changing the existing test cases, can we leave the listener and existing MeshGatewayRoutes untouched?